### ISSUE TITLE
Monitors_to_dataframe() NaN's

### ISSUE THREAD

**Issue by jd-lara** [dss-extensions/OpenDSSDirect.py#12]
Please provide the following information before submitting your issue. Thank you!

- Python 3.6
- Architecture x64
- MaxOSX
-  OpenDSS direct 0.2.14

The command ```dss.utils.monitors_to_dataframe()``` return a DataFrame with NaNs. 

The monitors are correct since running the export command return valid CSV files.

**Comment by kdheepak** [comment]
Thanks for reporting, I'll look into this.

**Comment by kdheepak** [comment]
Can you check if you are still experiencing this in v0.3.0?

**Comment by PMeira** [comment]
I ran the test case from [123Bus/SolarRamp.DSS](https://github.com/PMeira/electricdss-tst/blob/master/Distrib/IEEETestCases/123Bus/SolarRamp.DSS) and found a little issue -- `get_float64_array` was used but not imported in `dss.Monitors`. Since it was a very simple fix, I tested locally and committed it to master.

After I added the missing import, the results seem OK. The byte-stream or the channels themselves are not included in the current version. Maybe the byte-stream was being misinterpreted before, resulting in NaNs.

We could add a similar function, e.g. `monitor_to_dataframe`, with returns a more complete dataframe for a single monitor, including all the channels with a proper header. I'll make a PR about this soon since I could see myself using it.
